common/client/models.go

```diff
@@ -0,0 +1,17 @@
+package client
+
+type ClientErrorType int
```

Comment:
These are not just errors, but response codes.
Successful, TransactionAlreadyKnown, etc are not errors.

Similar to how http says return codes, can we rename this to SendTxReturnCode?